.github/ISSUE_TEMPLATE/user-story.md

```diff
@@ -0,0 +1,26 @@
+---
+name: User story
+about: Provide a user story to help the app's development.
+
+---
+
+## Description
+As a [USER],
+I need [TO DO THIS],
+so that I can [ACCOMPLISH THAT].
+
+## Mocks
+[INSERT RELEVANT PNG FILE]
+
+## Acceptance Criteria
+### Update [Required]
+- [ ] [LIST ITEMS]
+### Enhancement to Update [Optional]
+- [ ] [LIST ITEMS]
+
+## Definition of Done
+- [ ] All of the required items are completed.
+- [ ] Approval by 1 mentor.
+
+## Estimation
+[INSERT NUMBER HERE] hours
```

Comment:
Also, how can a user know how much time adding a particular feature would take? I'd remove this or change to `[INSERT NUMBER HERE] hours or [REMOVE THIS IF YOU DON'T KNOW]`.

Reply:
good point @bartekpacia . estimation in time is subjective as well, I probably cannot estimate issues that involve changing flutter code. For this PR I am fine with either, for the future this is a good issue to bring to @anitab-org/content-team

Reply:
+1 @bartekpacia 
I myself had trouble estimating time periods; it is just quite difficult to say how much this fix/enhancement would take (specially if it requires some sort of research).